contrib/json.arrai

```diff
@@ -0,0 +1,38 @@
+# Helper functions for working with JSON data.
+
+# Returns a "simplified" structure, replacing decoded tuples with their internal values.
+# Intended for cases in which the types of null values is not interesting.
+let rec simplify = \j
+    cond j {
+        [...]: j >> simplify(.),
+        (:a): simplify(a),
+        (:s): simplify(s),
+        (): {},
+        _: cond {
+            j < 99999999999999999999: j,
```

Comment:
is this meant to check if `j` is a number?

Reply:
Yeah. I forget now what the problem was, but I couldn't figure out how else to do it.

Reply:
Added a comment as well.